contracts/token/ERC721/IERC721Receiver.sol

```diff
@@ -1,11 +1,11 @@
-pragma solidity ^0.6.0;
+pragma solidity ^0.6.2;
```

Comment:
I don't think this bump is necessary, is it? We want to keep the previous one to avoid a breaking change for our users.

Reply:
@frangio  I did it for consistency as all the other interfaces for ERC721 are in that version (^0.6.2) but can certainly remove (it would be different than the other Interfaces). Just let me know! Thanks!

Reply:
Hm, I've forgot why the other interfaces have that pragma. But we should still keep this one as it is now if it's not necessary to bump.

Reply:
It's because the other ones inherit from each other, and interface inheritance was only added in 0.6.2.